src/views/portfolio/vulnerabilities/VulnerabilityCreateVulnerabilityModal.vue

```diff
@@ -548,6 +842,19 @@ export default {
       }
       return null;
     },
+    generateOwaspVector: function() {
+      if (this.owaspVector.sl != null && this.owaspVector.m != null && this.owaspVector.o != null
+        && this.owaspVector.s != null && this.owaspVector.ed != null
+        && this.owaspVector.ee != null &&this.owaspVector.a != null && this.owaspVector.id != null
+        && this.owaspVector.lc != null && this.owaspVector.li != null && this.owaspVector.lav != null
+        && this.owaspVector.lac != null && this.owaspVector.fd != null && this.owaspVector.rd != null
+        && this.owaspVector.nc != null && this.owaspVector.pv != null) {
+        let vector = `SL:${this.owaspVector.sl}/M:${this.owaspVector.m}/O:${this.owaspVector.o}/S:${this.owaspVector.s}/ED:${this.owaspVector.ed}/EE:${this.owaspVector.ee}/A:${this.owaspVector.sl}/ID:${this.owaspVector.id}/LC:${this.owaspVector.lc}/LI:${this.owaspVector.li}/LAV:${this.owaspVector.lav}/LAC:${this.owaspVector.lac}/FD:${this.owaspVector.fd}/RD:${this.owaspVector.rd}/NC:${this.owaspVector.nc}/PV:${this.owaspVector.pv}`;
```

Comment:
Skill Level is currently submitted as Awareness value.

```suggestion
        let vector = `SL:${this.owaspVector.sl}/M:${this.owaspVector.m}/O:${this.owaspVector.o}/S:${this.owaspVector.s}/ED:${this.owaspVector.ed}/EE:${this.owaspVector.ee}/A:${this.owaspVector.a}/ID:${this.owaspVector.id}/LC:${this.owaspVector.lc}/LI:${this.owaspVector.li}/LAV:${this.owaspVector.lav}/LAC:${this.owaspVector.lac}/FD:${this.owaspVector.fd}/RD:${this.owaspVector.rd}/NC:${this.owaspVector.nc}/PV:${this.owaspVector.pv}`;
```

Reply:
Done